light-client/ibc-client-starknet/src/client_state/execution.rs

```diff
@@ -82,7 +86,7 @@ where
                 new_consensus_state.into(),
             )?;
 
-            Ok(vec![latest_height])
+            Ok(vec![current_height])
```

Comment:
this was incorrect before. should be the height at which the consensus state was updated.

ref: the `ContractResult` struct at wasm-008 spec: https://github.com/cosmos/ibc/blob/main/spec/client/ics-008-wasm-client/README.md#contract-sudo